server/views/pages/activities/manage-allocations/allocateMultiplePeople/reviewUploadPrisonerList.njk

```diff
@@ -58,28 +71,154 @@
         {
           text: "Name",
           classes: "govuk-!-display-none-print",
-          attributes: { "aria-sort": "ascending", "data-qa": "prisoner-summary" }
+          attributes: { "aria-sort": "ascending", "data-qa": "unallocated-inmate-name" }
         },
         {
           text: "Cell location",
           classes: "govuk-!-display-none-print",
-          attributes: { "aria-sort": "none", "data-qa": "prisoner-cell-location"}
+          attributes: { "aria-sort": "none", "data-qa": "unallocated-inmate-cell-location"}
         },
         {
-          text: "Prison number",
-          classes: "print-only"
+          text: "Non-associations",
+          classes: "govuk-!-display-none-print",
+          attributes: { "aria-sort": "none", "data-qa": "unallocated-inmate-non-assoications"}
+        },
+        {
+          text: "Current allocations",
+          classes: "govuk-!-display-none-print",
+          attributes: { "aria-sort": "none", "data-qa": "unallocated-inmate-current-allocations"}
         },
         {
-          html: '<span class="govuk-visually-hidden">Actions</span>',
```

Comment:
Might actually need to keep this in for accessibility reasons - all the columns need a header. So we just need something with the hidden class